test/fast/errors-test.ts

```diff
@@ -40,26 +40,29 @@ describe('detects errors', () => {
 
     expect(result).toHaveGitError(GitError.BranchAlreadyExists)
   })
-  it('UnsafeDirectory', async () => {
-    const repoName = 'branch-already-exists'
-    const path = await initialize(repoName)
+  if (process.platform === 'win32') {
+    // maybe this is needed because GIT_TEST_ASSUME_DIFFERENT_OWNER is only a git for windows thing?
```

Comment:
Does this make any sense? Cause the test run fine on my mac without this check :/

Reply:
No it's a thing on macOS as well and it fails as it should when I skip setting GIT_TEST_ASSUME_DIFFERENT_OWNER on my machine 😞 

Seems this was working in https://github.com/desktop/dugite/pull/472 and started failing when https://github.com/desktop/dugite/pull/473 was merged but I can't for the life of me understand why. Gonna do some digging but the goal needs to be to fix the default branch here so that we can merge this without disabling the test

Reply:
>  the goal needs to be to fix the default branch here

Totally agree... But I was completely perplexed, I could find the reference to this in the git for windows repo but not regular git repo.